Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp

```diff
@@ -0,0 +1,183 @@
+
+  // Utility
+  auto* utility_box = new QGroupBox(tr("Utility"));
+  auto* utility_layout = new QGridLayout();
+  utility_box->setLayout(utility_layout);
+
+  m_dump_textures = new GraphicsBool(tr("Dump Textures"), Config::GFX_DUMP_TEXTURES);
+  m_load_custom_textures = new GraphicsBool(tr("Load Custom Textures"), Config::GFX_HIRES_TEXTURES);
+  m_prefetch_custom_textures =
+      new GraphicsBool(tr("Prefetch Custom Textures"), Config::GFX_CACHE_HIRES_TEXTURES);
+  m_use_fullres_framedumps = new GraphicsBool(tr("Full Resolution Frame Dumps"),
+                                              Config::GFX_INTERNAL_RESOLUTION_FRAME_DUMPS);
+  m_dump_efb_target = new GraphicsBool(tr("Dump EFB Target"), Config::GFX_DUMP_EFB_TARGET);
+  m_enable_freelook = new GraphicsBool(tr("Free Look"), Config::GFX_FREE_LOOK);
+  m_dump_use_ffv1 = new GraphicsBool(tr("Frame Dumps Use FFV1"), Config::GFX_USE_FFV1);
+
+  utility_layout->addWidget(m_dump_textures, 1, 1);
+  utility_layout->addWidget(m_load_custom_textures, 1, 2);
+  utility_layout->addWidget(m_prefetch_custom_textures, 2, 1);
+  utility_layout->addWidget(m_use_fullres_framedumps, 2, 2);
+  utility_layout->addWidget(m_dump_efb_target, 3, 1);
+  utility_layout->addWidget(m_enable_freelook, 3, 2);
+#if defined(HAVE_FFMPEG)
+  utility_layout->addWidget(m_dump_use_ffv1, 4, 1);
+#endif
+
+  // Misc.
+  auto* misc_box = new QGroupBox(tr("Misc."));
+  auto* misc_layout = new QGridLayout();
+  misc_box->setLayout(misc_layout);
+
+  m_enable_cropping = new GraphicsBool(tr("Crop"), Config::GFX_CROP);
+  m_enable_prog_scan = new QCheckBox(tr("Enable Progressive Scan"));
+  m_borderless_fullscreen =
+      new GraphicsBool(tr("Borderless Fullscreen"), Config::GFX_BORDERLESS_FULLSCREEN);
+
+  misc_layout->addWidget(m_enable_cropping, 1, 1);
+  misc_layout->addWidget(m_enable_prog_scan, 1, 2);
+#ifdef _WIN32
+  misc_layout->addWidget(m_borderless_fullscreen, 2, 1);
+#endif
+
+  AddWidget(debugging_box);
+  AddWidget(utility_box);
+  AddWidget(misc_box);
+
+  AddDescriptionBox();
+}
+
+void AdvancedWidget::ConnectWidgets()
+{
+  connect(m_enable_prog_scan, &QCheckBox::toggled, this, &AdvancedWidget::SaveSettings);
+  connect(m_load_custom_textures, &QCheckBox::toggled, this, &AdvancedWidget::SaveSettings);
+}
+
+void AdvancedWidget::LoadSettings()
+{
+  m_enable_prog_scan->setChecked(Settings::Instance().GetProgressiveScanEnabled());
+}
+
+void AdvancedWidget::SaveSettings()
+{
+  Settings::Instance().SetProgressiveScanEnabled(m_enable_prog_scan->isChecked());
+  m_prefetch_custom_textures->setEnabled(g_Config.bHiresTextures);
+}
+
+void AdvancedWidget::OnBackendChanged(const QString&)
+{
+  m_use_fullres_framedumps->setEnabled(g_Config.backend_info.bSupportsInternalResolutionFrameDumps);
+}
+
+void AdvancedWidget::AddDescriptions()
+{
+  static const QString TR_WIREFRAME_DESCRIPTION =
+      tr("Render the scene as a wireframe.\n\nIf unsure, leave this unchecked.");
```

Comment:
I'm fairly certain you need to annotate these with [QT_TR_NOOP()](https://doc.qt.io/qt-5/qtglobal.html#QT_TR_NOOP) and use tr() at the usage site of them, in order for translations to work properly (see Qt's translation [guide](https://doc.qt.io/qt-5/i18n-source-translation.html)). Local statics are initialized on program startup, so runtime translation wouldn't work at the point of variable initialization.